lib/vmpooler/api/v1.rb

```diff
@@ -28,6 +28,60 @@ def need_token!
       validate_token(backend)
     end
 
+    def alias_deref(hash)
+      newhash = {}
+
+      hash.each do |key, val|
+        if backend.exists('vmpooler__ready__' + key)
+          newhash[key] = val
+        else
+          if Vmpooler::API.settings.config[:alias][key]
+            newkey = Vmpooler::API.settings.config[:alias][key]
+            newhash[newkey] = val
+          end
+        end
+      end
+
+      newhash
+    end
+
+    def checkout_vm(template, result)
```

Comment:
This is basically the same content that was here before, just in a different spot in the file, correct?